addons/pos_restaurant/static/src/overrides/components/product_screen/product_screen.xml

```diff
@@ -5,23 +5,30 @@
         <xpath expr="//button[hasclass('pay-button')]" position="replace">
             <button
                 t-if="this.swapButton"
-                class="btn-switchpane btn btn-primary flex-fill rounded-0 fw-bolder py-3"
+                class="btn-switchpane btn flex-fill rounded-0 fw-bolder py-3"
                 t-on-click="submitOrder"
-                t-att-class="{'primary': primaryPayButton(), 'secondary': !primaryPayButton()}">
+                t-attf-class="{{ primaryOrderButton ? 'btn-primary' : 'btn-secondary' }}">
                 <!-- Replace the payment button by the order button -->
                 <span class="fs-1 d-block">Order</span>
-                <span><t t-esc="total"/></span>
+                <span><t t-esc="nbrOfChanges"/> changes</span>
             </button>
             <t t-else="">
-                <button 
-                    class="btn-switchpane btn btn-primary flex-fill rounded-0 fw-bolder py-3" 
-                    t-att-class="{'primary': primaryPayButton(), 'secondary': !primaryPayButton()}"
+                <button
+                    class="btn-switchpane btn flex-fill rounded-0 fw-bolder py-3"
+                    t-attf-class="{{ primaryPayButton ? 'btn-primary' : 'btn-secondary' }}"
                     t-on-click="() => currentOrder.pay()">
                     <span class="fs-1 d-block">Pay</span>
                     <span><t t-esc="total" /></span>
                 </button>
             </t>
         </xpath>
+        <xpath expr="//button[hasclass('review-button')]" position="replace">
+            <button class="btn-switchpane btn w-50 rounded-0 fw-bolder review-button" t-attf-class="{{ primaryReviewButton ? 'btn-primary' : 'btn-secondary' }}" t-on-click="switchPane">
+                <span class="fs-1 d-block">Payment</span>
+                <span t-if="this.swapButton"><t t-esc="total" /></span>
+                <span t-else=""><t t-esc="items"/> items</span>
+            </button>
+        </xpath>
```

Comment:
I don't understand why we need to change "review" to "payment". This change doesn't seem to be important.

Reply:
It was in discussion with Alain that we decided that